samples/dect/dect_phy/dect_shell/src/dect/mac/dect_phy_mac_nbr_bg_scan.c

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/sys/byteorder.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <modem/nrf_modem_lib.h>
+
+#include "desh_print.h"
+
+#include "dect_app_time.h"
+#include "dect_common.h"
+#include "dect_common_utils.h"
+#include "dect_common_settings.h"
+
+#include "dect_phy_api_scheduler.h"
+
+#include "dect_phy_ctrl.h"
+#include "dect_phy_mac_nbr.h"
+#include "dect_phy_mac_nbr_bg_scan.h"
+
+struct dect_phy_mac_nbr_bg_scan_metrics_data {
+	uint32_t scan_started_ok_count;
+	uint32_t scan_start_fail_count;
+	uint32_t scan_info_updated_count;
+	uint32_t scan_info_time_shift_updated_count;
+	int64_t scan_info_time_shift_last_value;
+};
+
+struct dect_phy_mac_nbr_bg_scan_data {
+	bool running;
+	struct dect_phy_mac_nbr_bg_scan_params params;
+	struct dect_phy_mac_nbr_bg_scan_metrics_data metrics;
+
+	uint64_t last_updated_rcv_time;
```

Comment:
If this is in milliseconds, Zephyr "standard" would be to use `int64_t` but I guess `uint64_t` is pretty much built into DESH already.

Reply:
This is in modem ticks. Maybe I could rename the variable accordingly.